nautobot_golden_config/models.py

```diff
@@ -867,14 +869,11 @@ class ConfigPlan(PrimaryModel):  # pylint: disable=too-many-ancestors
         to="extras.JobResult",
         on_delete=models.CASCADE,
         related_name="config_plan",
-        null=False,
-        blank=False,
```

Comment:
I am still not sure about this change.

We decided to generate config plans with job-helper, however we ended-up:

- disabling CSV imports of config plans
- disabling API creation of config plans

Reply:
Nothing has functionally changed

<img width="938" alt="image" src="https://github.com/nautobot/nautobot-plugin-golden-config/assets/9260483/2c76430f-c454-4c13-8712-cb9c833b0d75">

Reply:
I was saying , that the ConfigPlan model makes the `jobID` mandatory - you cannot have ConfigPlan without having a Job first. 
The only one point I am making today is that we can possibly have more ways of populating this in the future - CSVs , APIs etc.